hls4ml/utils/config.py

```diff
@@ -284,6 +285,7 @@ def config_from_pytorch_model(
 
     Args:
         model: PyTorch model
+        input_shape (list): The shape of the input tensor. First element is the batch size, needs to be None
```

Comment:
We have a nice opportunity here to get rid of the first `None` in the shape. Just ask users to not put it, but then put it in ourselves :wink:

Reply:
Good idea, that was always very ugly. Done.

Reply:
In general I think we need to be careful about the batch dimensions. For ONNX it's usually just 1 (but can really be any value, e.g. 10), not None. I think we try to get rid of the batch dimension pretty quickly.